exporters/dsexporter/dsexporter.go

```diff
@@ -1,39 +1,115 @@
 package main
 
 import (
+	"context"
+	"encoding/json"
 	"fmt"
 	"net/http"
+	"os"
+	"strings"
 
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
 )
 
+const service = "https://console-openshift-console.apps.stone-prd-rh01.pg1f.p1.openshiftapps.com/k8s/ns/appstudio-grafana/services"
```

Comment:
This is going to be different for each cluster. No?

Reply:
Yes, it's different for each cluster

Reply:
We discussed this in a call. The `service` label should be a constant descriptor for the service under evaluation and it should not be cluster scoped. In this case I think we could use `grafana` since we're checking the availability of grafana-related configuration.